### ISSUE TITLE
Whatsapp integration - fails to find webhook

### ISSUE THREAD

**Issue by iReserveApps** [freescout-helpdesk/freescout#2821]
Hi, 

We are trying to setup Whatsapp via Twillio
We followed the onboarding process. Number is active, Twillio account is fully onboarded. 
Activated the Whatsapp module in Freescout and entered the credentials for one of the mailboxes. 
However, Twillio ends in error, as the copied webhook is not reachable. 

<img width="1340" alt="image" src="https://user-images.githubusercontent.com/61552383/224687086-67366c54-96e5-4204-8e39-f9006614e776.png">

When we try to access that webhook via testing (postman, restclient) it indeed gives a 404 status request. 
Perhaps something went wrong during installing the module. 

In log we see two statements

Mar 13, 12:27:27 | [WhatsApp Webhook - 1msg.io] (Info)  Empty user.
-- | --
Mar 13, 12:22:41 | [WhatsApp Webhook - Twilio] (Info)  Incorrect Account SID received in webhook:

1msg.io is NOT selected. 
We double checked the Account SID, but that is the same as being send from Twillio. 

Where can we see the details of this log, as it might be we are missing information?

**Comment by iReserveApps** [comment]
We have found the cause of the issue. 
Two items were not updated. The customer class (cannot find the function byPhone) and the rpt_ready column in the module Reports. This was giving errors and hence throwing a fatal error. 

We have posted other issues about local and dev environments in combination with an update which fails. 
Really struggling with the upkeep, which is unfortunate as we are so happy about the app itself. 
So basically: updates in production environment via the GUI fails. Which causes us to enforce a local environment + FTP changes. But that itself is not enough. DB changes need to happen as well. 

We need a guideline how to 
- apply updates to FS + modules on a local environment
- Then we test (no guideline here needed)
- How to sync the prod environment with the changes.
